extensions/statsd-emitter/README.md

```diff
@@ -0,0 +1,19 @@
+## introduction
+
+This extension emits druid metrics to a StatsD server.
+
+## configuration
+
+To enable set:
+`druid.emitter=statsd`
+
+StatsD endpoint is set by:
+`druid.emitter.statsd.hostname` and `druid.emitter.statsd.port`
+
+Additionally, you must provide a mapping from Druid metric name, to StatsD metric type via:
```

Comment:
"must" makes it seem that this config is mandatory, however from the code it appears that if its not provided, defaults are used. I think that should be explained.